client/web/src/components/panels/LayerTree.vue

```diff
@@ -20,7 +20,13 @@
 					<div class="layer-visibility">
 						<IconButton :icon="layer.visible ? 'EyeVisible' : 'EyeHidden'" @click="toggleLayerVisibility(layer.path)" :size="24" :title="layer.visible ? 'Visible' : 'Hidden'" />
 					</div>
-					<div class="layer">
+					<div
+						class="layer"
+						:class="{ selected: layer.layer_data.selected }"
+						@click.shift.exact="handleShiftClick(layer)"
+						@click.alt.exact="handleControlClick(layer)"
```

Comment:
Add another line:
```
						@click.ctrl.exact="handleControlClick(layer)"
```

Someday we'll properly deal with platform-specific keybinding differences with Mac.